src/composables/screeningsList.js

```diff
@@ -0,0 +1,21 @@
+import { useGetters } from "vuex-composition-helpers";
+import { getDate } from "@/helpers"
+
+export const screeningsList = () => {
```

Comment:
A minor thing, very minor but as a convention composables usually have a name like `useSomething` like `useMouse`, `useGetters`, `useRouter`. So we could name this as `useScreeningsList` 💡